backend/src/domain/models/Game.js

```diff
@@ -60,6 +61,7 @@ export class Game {
 
 
     for (let i=0; i < this.swipeDeck.length; i++) {
+      // notifies users of round change
```

Comment:
And also this. Same goes for the remaining of the PR. Not to important, but would just be nice to keep things cleaner.